QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_002015	$With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. The best place to hide a tree is in a forest. {nl}Ruko Plateau and the Plateau of Rex… I'm so close I can almost touch it. {nl}I'm confident the Goddess sent him to me. May the blessing of the Goddess always be with us!
+QUEST_20150511_002016	$Some poles unlock the tomb and some altars lock it. {nl}The magical energy that surrounds the tomb moves by the key. {nl}But none can be that key. {nl}We are not chosen.
+QUEST_20150511_002017	$The curse disturbs the rules of the universe that are being controlled by the Goddess with human greed.{nl}These rules in the human world get influenced by the personalities of each human, their abilities and their potential power. {nl}The cases when these abilities get released depend on the conditions above.
+QUEST_20150511_002018	$According to the old writing that is left on the great cathedral, that is using a small part of the Goddess' abilities secretly.{nl}Some people say that after seducing weak minds from humans, the demons used them for the source of the curse. {nl}May the Goddess bless!
+QUEST_20150511_002019	$There are two ways to release the curse: {nl}Remove the curse using a greater power or to eliminate the caster of the curse. {nl}
+QUEST_20150511_002020	$Removing the curse with a greater power: {nl}This way depends on whether the power that overwhelms the curse comes from good or evil and from darkness or light.{nl}{nl}The most important factor for eliminating the caster: {nl}Even after eliminating the caster, the curse may not be removed due to other factors.{nl}{nl}I have no doubt that the power of the Goddess is stronger.
+QUEST_20150511_002021	$There are mainly two ways to spread the curse. {nl}First, using equipment and uttering simple words. It sounds easy but it is, in fact, the hardest way. {nl}Therefore, from the example of the Pilgrim Path, we mainly use the cursed ones as the mediums to spread the curse.
+QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous influence on the mind of the victim. Their ego usually can't endure the feeling of guilt this causes. {nl}It seems the magic that was used on the cursed city of Ruklys seems to be of a higher level than the one used on Naktis. We need to look into it more.
+QUEST_20150511_002023	$The Completed Research Materials of Varkis
+QUEST_20150511_002024	$The First Research Material{nl}-{nl}We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to make it.{nl}I believe only this place can answer the question I've had in my mind for a long time.
```

Comment:
These lines seems to duplicate some of the above lines. Please see my comments on those lines and apply them here as well.